tests/sentry/api/endpoints/test_organization_spans_fields.py

```diff
@@ -89,6 +86,38 @@ def do_request(self, query=None, features=None, **kwargs):
                 **kwargs,
             )
 
+    def test_tags_list(self):
```

Comment:
are there two test_tags_list ?

Reply:
Yeah one is inheriting from `OrganizationSpansTagsEndpointTest` while the other is that class